iphone/cli/commands/_build.js

```diff
@@ -5768,14 +5768,29 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
 						imageNameRegExp = /^(.*?)(-dark)?(@[23]x)?(~iphone|~ipad)?\.(png|jpg)$/;
 
 					Object.keys(imageAssets).forEach(function (file) {
+						const directories = file.split('/');
+						let newPath = '';
+						for (let i = 0; i < directories.length - 1; i++) {
+							newPath = newPath + '/' + directories[i];
+
+							writeAssetContentsFile.call(this, path.join(assetCatalog, newPath, 'Contents.json'), {
```

Comment:
Are you sure you mean to create a separate `Contents.json` file in each directory?

Also, this code will recreate the file for higher up directories. In other words `foo/bar/baz.png` and `foo/wiz/pow.png` will create `foo/Contents.json` twice. Seems unnecessary.

Reply:
1. Yes, Content.json is required in each directory.
2. I believe condition "if (!fs.existsSync(dest) || contents !== fs.readFileSync(dest).toString())" in function "writeAssetContentsFile(dest, json)" will ensure not to write Content.js json twice in same directory.

Reply:
Ah, I see now. Make sense. Thanks!